app/controllers/user_authentication_controller.rb

```diff
@@ -43,10 +43,10 @@ def create
     @user.password_confirmation = params.fetch("query_password_confirmation")
     @user.user_type = params.fetch("query_user_type")
     @user.name = params.fetch("query_name")
-    @user.avatar = params.fetch("query_avatar")
-    @user.burritos_count = params.fetch("query_burritos_count")
-    @user.ratings_count = params.fetch("query_ratings_count")
-    @user.restaurants_count = params.fetch("query_restaurants_count")
+    @user.avatar = ""
+    @user.burritos_count = 0
+    @user.ratings_count = 0
+    @user.restaurants_count = 0
```

Comment:
If you want these columns to have a default value of 0, you can specify it in [the migration for the User table]. Then you won't need to set these values here.